pages/common/xml.md

```diff
@@ -0,0 +1,29 @@
+# xml
+
+> XMLStarlet Toolkit: Query, edit, check, convert and transform XML documents.
+> Subcommands: `canonic`, `edit`, `elements`, `escape`, `unescape`, `format`, `list`, `pyx`, `depyx`, `select`, `transform`, or `validate`.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Execute a subcommand with input from a file or URI and using standard output:
+
+`xml {{subcommand}} {{options}} {{path/to/input.xml|URI}}`
+
+- Execute a subcommand using standard input and standard output:
+
+`xml {{subcommand}} {{options}}`
+
+- Execute a subcommand with input from a file or URI and output to a file:
+
+`xml {{subcommand}} {{options}} {{path/to/input.xml|URI}} >{{path/to/output}}`
```

Comment:
Same as above

Reply:
The syntax shown is a useful summary for 10 of the 12 subcommands. It's usefulness as a summary is not decreased by the existence of exceptions. Pointing out the exceptions here would distract from understanding.